templates/galaxy/webhooks/subdomain_switcher/script.js.j2

```diff
@@ -0,0 +1,82 @@
+(() => {
+
+  $(document).ready( function() {
+
+    const SUBDOMAINS = [
+      {
+        label: "Base site",
+        url: "https://{{ galaxy_instance_hostname }}",
+      },
+      {% for subdomain in galaxy_themes_subdomains %}
```

Comment:
Those are defined in group_vars/sn06/subdomains.yml ... I hope they are valid here.